keras_cv/models/resnet_v2.py

```diff
@@ -355,10 +374,21 @@ def ResNetV2(
 
     if weights is not None:
         model.load_weights(weights)
+    # Set this private attribute for recreate backbone model with outputs at each of the
+    # resolution level.
+    model._backbone_level_outputs = stack_level_outputs
 
     return model
 
 
+def create_backbone_model(model):
```

Comment:
maybe add some documentation here, how we imagine users to use this?

Reply:
Done. Changed to a new interface.